.gitignore

```diff
@@ -5,6 +5,7 @@ bid/development-bundle
 bid/backup/
 content/cache/*
 dbconfig.inc
+dbconfig.py
```

Comment:
Now I wonder if this should be json or similar instead, and have python/php just read it directly?

Either way, can you add a dbconfig.py.sample ?